Sources/CoreData/Combine/DataRequest+Combine.swift

```diff
@@ -0,0 +1,69 @@
+//
+// AppwiseCore
+// Copyright © 2022 Appwise
+//
+
+import Alamofire
+import CocoaLumberjack
+import Combine
+import CoreData
+
+@available(iOS 13, *)
+public extension DataRequest {
+	@discardableResult
+	func responseInsert<T: Insertable>(
+		of type: T.Type = T.self,
+		db: DB = DB.shared,
+		queue: DispatchQueue = .main,
+		jsonOptions: JSONSerialization.ReadingOptions = .allowFragments,
+		jsonTransformer: @escaping (Any) throws -> Any = { $0 },
+		contextObject: Any? = nil
+	) -> AnyPublisher<DataResponse<T, Error>, Never> {
+		let context = db.newBackgroundContext()
```

Comment:
Can't this just wrap the call to non-combine `responseInsert`? (like you did with the others)